### PR TITLE
Feature: add transition to and from evidence question page

### PR DESCRIPTION
**Pull request by akaJSaunders**
### JIRA link

https://companieshouse.atlassian.net/browse/LFA-1280

### Change description

Using "Continue" on "Do you want to add documents..." screen, if no validation messages were shown and the user selected “No”, takes the user to the check-answers screen.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/views/evidence-upload.njk [hunk 1]
```diff
@@ -10,6 +10,21 @@
...
+      outline: none;
+      background: none;
+      cursor: pointer;
+      color: #0000EE;
+      padding: 0;
+      text-decoration: underline;
+      font-family: inherit;
+      font-size: inherit;
+    }
+  </style>
```
Comment (akaJSaunders) [inline src/views/evidence-upload.njk:26]: This is temporary for development process

File: test/controllers/EvidenceQuestionController.test.ts [hunk 2]
```diff
@@ -55,26 +66,26 @@ describe('EvidenceQuestionController', () => {
 
         describe('when answer is NO', () => {
             it('should refresh page', async () => {
```
Comment (geek-soft) [inline test/controllers/EvidenceQuestionController.test.ts:68]: Test desc is invalid now.

File: test/controllers/EvidenceQuestionController.test.ts [hunk 3]
```diff
@@ -55,26 +66,26 @@ describe('EvidenceQuestionController', () => {
...
                     .expect(response => {
                         expect(response.status).to.be.equal(MOVED_TEMPORARILY);
-                        expect(response.get('Location')).to.be.equal(EVIDENCE_QUESTION_URI);
+                        expect(response.get('Location')).to.be.equal(CHECK_YOUR_APPEAL_PAGE_URI);
                     });
             });
         });
 
         describe('when answer is YES', () => {
             it('should refresh page', async () => {
```
Comment (geek-soft) [inline test/controllers/EvidenceQuestionController.test.ts:81]: Test name is outdated now

File: src/views/evidence-upload.njk [hunk 4]
```diff
@@ -85,6 +100,11 @@
       <p>
         <a href="{{ navigation.forward.href }}" class="govuk-link">Continue without adding documents</a>
       </p>
+      <form method="post">
```
Comment (geek-soft) [inline src/views/evidence-upload.njk:103]: I suggest that work is moved to separate PR while this PR focuses only on linking evidence question page.
